### PR TITLE
Add support for RBAC mappings

### LINKED ISSUE #318: Support for RBAC mappings
It would be nice to have support for RBAC mappings (I'll probably take a swing at this, it's something that I need right now).

### PR DESCRIPTION
**Pull request by mamercad**
Fixes #318.

### CODE REVIEW

File: roles/StackStorm.st2/defaults/main.yml [hunk 1]
```diff
@@ -54,6 +54,19 @@ st2_ldap:
 # The schema for roles and assignments follow the exact schema definition
 # define in https://docs.stackstorm.com/latest/rbac.html#defining-roles-and-permission-grants
 # and https://docs.stackstorm.com/latest/rbac.html#defining-user-role-assignments
+# Mappings are pushed as YML files to /opt/stackstorm/rbac/mappings/
+# The schema for mappings takes on the following format:
+#   - The well-defined "name" key is used as the filename, e.g., the exmaple below creates /opt/stackstorm/rbac/mappings/domain-admins.yaml
```
Comment (amanda11) [inline roles/StackStorm.st2/defaults/main.yml:59]:
```suggestion
#   - The well-defined "name" key is used as the filename, e.g., the example below creates /opt/stackstorm/rbac/mappings/domain-admins.yaml
```

File: roles/StackStorm.st2/templates/rbac_mappings/mappings.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+
+{{ item.mapping | to_nice_yaml(2) }}
```
Comment (amanda11) [inline roles/StackStorm.st2/templates/rbac_mappings/mappings.yml.j2:3]: In most of our usage we also pipe to | indent(2) - perhaps that would be useful here too?

**Review by amanda11 — COMMENTED** [review]
LGTM - thanks for the contribution. Just a couple of minor comments.

**Comment by mamercad** [comment]
Err; I think `indent(2)` only indents the first line ...

**Comment by mamercad** [comment]
My deployment is breaking now (as far as RBAC mappings go), here's one of the diffs:

```
-group: redacted
-roles:
-- admin
+  group: redacted
+  roles:
+  - admin
```

**Comment by mamercad** [comment]
After reverting `indent(2)`, my deployment is happy again.

**Review by amanda11 — APPROVED** [review]
Brilliant, thanks for the contribution.

**Pull request merged**